packages/kbn-optimizer/tsconfig.json

```diff
@@ -1,10 +1,18 @@
 {
   "extends": "../../tsconfig.base.json",
   "compilerOptions": {
-    "tsBuildInfoFile": "../../build/tsbuildinfo/packages/kbn-optimizer"
+    "incremental": false,
+    "outDir": "target",
+    "declaration": true,
+    "declarationMap": true,
+    "sourceMap": true,
+    "sourceRoot": "../../../../packages/kbn-optimizer/src"
```

Comment:
I was testing the imports of `@kbn/optimizer` and was unable to get back to the source, or target for that matter - but others did work.

Reply:
Can you try again after rebootstrapping? I've tried it locally it worked for me.